scripts/ci/kubernetes/minikube/start_minikube.sh

```diff
@@ -74,6 +71,7 @@ fi
 export PATH="${_MY_DIR}/bin:$PATH"
 
 if [[ "${USE_MINIKUBE_DRIVER_NONE:-}" = "true" ]]; then
+  # TODO: Change this it should be Travis-independent
```

Comment:
I don't see anything travis specific here - the comment is about it, but there are no travis specific commands or envs in use.

Reply:
See the comment 2 lines below - this is a hack for minikube with none driver to run it on Travis. I hope we can finally (with @gerardo help) make it CI-independent (and then we should delete the hack).

Reply:
This is already ci independent though - driver none is useful whenever we are running inside docker, no?

Reply:
I am not sure - I think in cloud build where docker-in-docker approach is used it will work differently. I was mostly referring to the URL and I can remove it. This part will likely change heavily if we go docker-compose way.

Reply:
Removed.